src/components/PokemonCards.js

```diff
@@ -0,0 +1,43 @@
+import React ,{useState,useEffect} from 'react';
+import axios from "axios";
+
+const PokemonCards = ({name}) => {
+    const [pokemonData,setPokemonData] = useState({});
+    async function fetchPokemon () {
+        try{
+            const result = await axios.get(`https://pokeapi.co/api/v2/pokemon/${name}`)
+            setPokemonData(result.data)
+
+        }catch (e){
+            console.error(e)
+        }
+    }
+    useEffect(()=>{
+        fetchPokemon()
+    },[])
+    return (
+        <>
+
+            {Object.keys(pokemonData).length > 0 &&
```

Comment:
De check kan hier dan wat simpeler
```suggestion
            {pokemonData &&
```